homeassistant/components/purpleair/__init__.py

```diff
@@ -71,6 +75,23 @@ def __init__(
             ATTR_LONGITUDE: self.sensor_data.longitude,
         }
 
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any]:
```

Comment:
With this property function added, remove setting of `self._attr_extra_state_attributes` above